### PR TITLE
netperf: fix TODO in inline documentation

### PR DESCRIPTION
**Pull request by mkoderer**

### CODE REVIEW

**Comment by mkoderer** [comment]
@gmarek could you have a look, since this TODO was from you?

File: network/benchmarks/netperf/nptest/nptest.go [hunk 1]
```diff
@@ -86,8 +86,7 @@ const (
 	netperfTest  = iota
 )
 
-// NetPerfRpc stores something
-// TODO: someone who knows what this all this does should update comments.
+// NetPerfRpc serivce that exposes RegisterClient and ReceiveOutput for clients
```
Comment (gonzolino) [inline network/benchmarks/netperf/nptest/nptest.go]: Typo: serivce -> service

**Pull request merged**